diff -u -r grafana-7.5.15/pkg/components/gtime/gtime_test.go grafana-7.5.15/pkg/components/gtime/gtime_test.go
--- grafana-7.5.15/pkg/components/gtime/gtime_test.go	2022-01-25 23:07:44.000000000 +0800
+++ grafana-7.5.15/pkg/components/gtime/gtime_test.go	2023-06-19 14:22:15.000000000 +0800
@@ -72,13 +72,13 @@
 
 func calculateDays() (int, int) {
 	now := time.Now().UTC()
-	currentYear, currentMonth, _ := now.Date()
+	currentYear, currentMonth, currentDay := now.Date()
 
 	firstDayOfMonth := time.Date(currentYear, currentMonth, 1, 0, 0, 0, 0, time.UTC)
 	daysInMonth := firstDayOfMonth.AddDate(0, 1, -1).Day()
 
-	t1 := time.Date(currentYear, 1, 1, 0, 0, 0, 0, time.UTC)
-	t2 := time.Date(currentYear+1, 1, 1, 0, 0, 0, 0, time.UTC)
+	t1 := time.Date(currentYear, currentMonth, currentDay, 0, 0, 0, 0, time.UTC)
+	t2 := time.Date(currentYear+1, currentMonth, currentDay, 0, 0, 0, 0, time.UTC)
 
 	daysInYear := int(t2.Sub(t1).Hours() / 24)
 
@@ -87,13 +87,13 @@
 
 func calculateDays5y() int {
 	now := time.Now().UTC()
-	currentYear, _, _ := now.Date()
+	currentYear, currentMonth, currentDay := now.Date()
 
 	var daysInYear int
 
 	for i := 0; i < 5; i++ {
-		t1 := time.Date(currentYear+i, 1, 1, 0, 0, 0, 0, time.UTC)
-		t2 := time.Date(currentYear+i+1, 1, 1, 0, 0, 0, 0, time.UTC)
+		t1 := time.Date(currentYear+i, currentMonth, currentDay, 0, 0, 0, 0, time.UTC)
+		t2 := time.Date(currentYear+i+1, currentMonth, currentDay, 0, 0, 0, 0, time.UTC)
 
 		daysInYear = daysInYear + int(t2.Sub(t1).Hours()/24)
 	}
